pom.xml

```diff
@@ -85,7 +85,12 @@
 		<dependency>
 			<groupId>commons-codec</groupId>
 			<artifactId>commons-codec</artifactId>
-			<version>1.10</version>
+			<version>1.11</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-lang3</artifactId>

```

Comment:
>commons-lang3< [](start = 15, length = 14)

did you register this one in the OSS tool as well?

Reply:
Yes